jiant/tasks/seq2seq.py

```diff
@@ -21,15 +22,19 @@
 )
 
 
-@register_task("seg_wix", rel_path="seg/wix/", max_targ_v_size=200)
+@register_task("seg_wix", rel_path="seg/wix/", max_targ_v_size=200, valid_metric="accuracy")
+@register_task("wmt14_en_de", rel_path="wmt14/en_de/", max_targ_v_size=40000, valid_metric="BLEU")
 class Seq2SeqTask(SequenceGenerationTask):
     """Sequence-to-sequence Task"""
 
-    def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
+    def __init__(self, path, max_seq_len, max_targ_v_size, name, valid_metric, **kw):
         super().__init__(name, **kw)
-        self.scorer2 = BooleanAccuracy()
+        if valid_metric == "accuracy":
+            self.scorer2 = BooleanAccuracy()
+            self.val_metric = "%s_accuracy" % self.name
+        else:  # for MT
+            self.scorer2 = BLEU(exclude_indices=set([0, 1, 2, 3]))  # PAD, SOS, EOS, UNK
```

Comment:
How/where do we verify that this index order is right? Won't this break if we use something like RoBERTa (which we'll have to for our paper) that has its own vocab?

Reply:
@HaokunLiu @pruksmhc Could you take a look at this?

Reply:
There might be a way to use the embedder https://github.com/nyu-mll/jiant/blob/9a47eea0f71021d8c714b4a455f6ad28793baa3c/jiant/pytorch_transformers_interface/modules.py and the _sep_id, _cls_id, _pad_id, and _unk_id variables.